lib/lorekeeper/multi_logger.rb

```diff
@@ -51,5 +51,9 @@ def call_loggers(method, *args, &block)
       # Then hope for the best taking the first value
       result.compact.uniq.first
     end
+
+    def method_missing(method, *args, &block)
```

Comment:
keep a little bit of metaprogramming to make our life easier. otherwise, we need to add 10+ methods more...

Reply:
now you understand my laziness